maths/monte_carlo.py

```diff
@@ -72,6 +82,29 @@ def area_under_line_estimator_check(iterations: int,
     print("Total error is ", abs(estimated_value - expected_value))
     print("******************")
 
+    return
+
+
+def pi_estimator_using_area_under_curve(iterations: int) -> None:
+    """
+    Area under curve y = sqrt(4 - x^2) where x lies in 0 to 2
+    is equal to pi
+    """
+
+    def function_to_integrate(x: float) -> float:
```

Comment:
This function needs doctests.

Reply:
Thanks for the suggestion. I have added. I am new to doctests. Please let me know if I need to update it.